### PR TITLE
MWPW-143085 - Assign body font styles to <ul> and <ol>

### PR DESCRIPTION
**Pull request by elan-tbx**
* Blocks that decorate their text using the `decorateBlockText` utility function did not account for \<ul\> and \<ol\> elements, which do not get nested inside paragraph tags (and therefore do not get a typography class applied to them/their parent). At the direction of Consonant, this PR aligns those elements to use the same typography styles as neighboring paragraph tags.

Resolves: [MWPW-143085](https://jira.corp.adobe.com/browse/MWPW-143085)

**Test URLs:**
Milo:
- Before: https://main--milo--adobecom.hlx.page/drafts/klee/test-media?martech=off
- After: https://ebartholomew-MWPW-143085-media-bullet--milo--adobecom.hlx.page/drafts/klee/test-media?martech=off

Bacom Pink:
- Before: https://main--bacom-pink--adobecom.hlx.page/products/experience-manager/forms/aem-forms?martech=off
- After: https://main--bacom-pink--adobecom.hlx.page/products/experience-manager/forms/aem-forms?milolibs=ebartholomew-MWPW-143085-media-bullet&martech=off

### CODE REVIEW

File: libs/utils/decorate.js [hunk 1]
```diff
@@ -60,11 +60,11 @@ export function decorateBlockText(el, config = ['m', 's', 'm'], type = null) {
         decorateIconArea(el);
       }
     }
-    const emptyPs = el.querySelectorAll(':scope p:not([class])');
+    const emptyPs = el.querySelectorAll('p:not([class]), ul:not([class]), ol:not([class])');
```
Comment (overmyheadandbody) [inline libs/utils/decorate.js:63]: Would it make sense to update the variable name, since it no longer exclusively refers to paragraph elements?

**Review by overmyheadandbody — APPROVED** [review]
One small nit

**Review by ryanmparrish — APPROVED** [review]
Looks good. I also agree that `emptyPs` const name, could be updated to `emptyEls` or something since the scope is broader now.

**Review by Sartxi — APPROVED** [review]
Approved, I also agree on the semantic variable name comments

**Comment by elan-tbx** [comment]
@ryanmparrish @Sartxi @overmyheadandbody Thanks y'all, `emptyEls` it is

**Comment by Blainegunn** [comment]
@elan-tbx let's get this one verified!

**Review by mokimo — APPROVED** [review]

**Pull request merged**